mmtrack/core/bbox/__init__.py

```diff
@@ -1,4 +1,8 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .iou_calculators import calculate_region_overlap
 from .transforms import bbox_cxcywh_to_x1y1wh, bbox_xyxy_to_x1y1wh, quad2bbox
```

Comment:
Please follow https://github.com/open-mmlab/mmdetection/pull/5781 to update the PR